internal/version/version.go

```diff
@@ -0,0 +1,56 @@
+package version
+
+import (
+	"errors"
+	"runtime/debug"
+)
+
+var (
+	Version string
```

Comment:
Use lowercase here because we do not want this accessed from outside this package.

```suggestion
	version string
```